### ISSUE TITLE
Docs Not building ?

### ISSUE THREAD

**Issue by shivance** [deepchem/deepchem#2975]
## 🐛 Bug

Apparently deepchem docs are not building or some issue exists.

In Coding Conventions Docs > Testing section 
[Docs](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html#testing-machine-learning-models) Link 
[Source](https://github.com/deepchem/deepchem/blob/master/docs/source/development_guide/coding.rst) Link

In the 4th line of commands to create test environment
source file says `env_test.yml` while in docs it is `env.test.yml`

**Comment by arunppsg** [comment]
Yeah, the build in readthedocs is failing - [ref](https://readthedocs.org/projects/deepchem/builds/). I think the issue is not with DeepChem. From build logs in [readthedocs](https://readthedocs.org/projects/deepchem/builds/17365860/), it raises an exception during the build with the following message:
```
Exception occurred:
  File "/home/docs/checkouts/readthedocs.org/user_builds/deepchem/envs/latest/lib/python3.7/site-packages/sphinx/ext/autodoc/__init__.py", line 82, in members_option
    return [x.strip() for x in arg.split(',')]
AttributeError: 'bool' object has no attribute 'split'
```
The exception raises in sphinx. Also the build is using python 3.7. Is there some way to switch to python 3.8 @rbharath ?

**Comment by rbharath** [comment]
I think we can definitely switch to python 3.8 for the build. Let's go ahead with that if it'll fix the build

**Comment by arunppsg** [comment]
Closing this since doc is getting built (fixed in #2985).
